### ISSUE TITLE
oik/shortcodes block - SelectControl is enormous

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik#195]
This has happened a few times before and then gone away. 
In the Site Editor beta, WordPress 5.9-RC3, the SelectControl in the main body is huge. It makes the editor unusable
The exact same code in the sidebar is fine. 

![image](https://user-images.githubusercontent.com/2474435/150107547-0de519b1-801b-4f58-9985-d1340b589f4d.png)

### Proposed solution

Either fix the CSS or remove the SelectControl from the main body.
There's a fairly good reason to remove it from the main body... it looks better in the editor.

**Comment by bobbingwide** [comment]
I've found the CSS that's causing the problem. 
![image](https://user-images.githubusercontent.com/2474435/150115118-6599fc9e-ac18-4007-8198-ae7373025332.png)

If I disable the inline style's `height: 100%` property for the encapsulating `<div>` then the select list's are all OK. 

```
<div data-wp-c16t="true" data-wp-component="Flex" class="components-flex components-select-control em5sgkm7 a--b-bbaa-bacdd-1cak69g em57xhy0"><div data-wp-c16t="true" data-wp-component="FlexItem" class="components-flex-item em5sgkm3 a--b-bbaa-bacdd-z5z9eh em57xhy0"><label data-wp-c16t="true" data-wp-component="Text" for="inspector-select-control-0" class="components-truncate components-text components-input-control__label em5sgkm4 a--b-bbaa-bacdd-3pxyi8 em57xhy0">Shortcode</label></div><div class="components-input-control__container a--b-bbaa-bacdd-1t6thiu em5sgkm6"><select class="components-select-control__input a--b-bbaa-bacdd-5c5yva e1mv6sxx1" id="inspector-select-control-0"><option value="bw_admin">bw_admin - Display the Admin contact name</option><option value="bw_alt_slogan">bw_alt_slogan - Alternative slogan</option><option value="bw_business">bw_business - Display your Business name</option><option value="bw_company">bw_company - Company name</option><option value="bw_contact">bw_contact - Primary contact name</option><option value="bw_contact_button">bw_contact_button - Contact form button</option><option value="bw_copyright">bw_copyright - Format a Copyright statement</option><option value="bw_directions">bw_directions - Display a 'Google directions' button.</option><option value="bw_domain">bw_domain - Display the domain name</option><option value="bw_email">bw_email - Email primary contact (formal)</option><option value="bw_emergency">bw_emergency - Emergency telephone number</option><option value="bw_fax">bw_fax - Fax number</option><option value="bw_formal">bw_formal - Formal company name</option><option value="bw_geo">bw_geo - Latitude and Longitude</option><option value="bw_login">bw_login - Display the login form or protected content</option><option value="bw_loginout">bw_loginout - Display the Login or Logout link</option><option value="bw_logo">bw_logo - Display the company logo</option><option value="bw_mailto">bw_mailto - Mailto (inline)</option><option value="bw_mob">bw_mob - Mobile phone number (inline)</option><option value="bw_mobile">bw_mobile - Mobile phone number (block)</option><option value="bw_more">bw_more - Read more button to progressively reveal content</option><option value="bw_parent">bw_parent - Display a link back to the parent page</option><option value="bw_pdf">bw_pdf - Display attached PDF files</option><option value="bw_portfolio">bw_portfolio - Display matched portfolio files</option><option value="bw_power">bw_power - Powered by WordPress</option><option value="bw_register">bw_register - Display a link to the Registration form, if Registration is enabled</option><option value="bw_skype">bw_skype - Skype name</option><option value="bw_slogan">bw_slogan - Primary slogan</option><option value="bw_tel">bw_tel - Telephone number (inline)</option><option value="bw_telephone">bw_telephone - Telephone number (block)</option><option value="bw_wpadmin">bw_wpadmin - Site: link to wp-admin</option><option value="clear">clear - Clear divs </option><option value="paypal">paypal - Paypal shortcodes</option><option value="OIK">OIK - Spells out the oik backronym</option></select><span class="components-input-control__suffix a--b-bbaa-bacdd-pvvbxf em5sgkm0"><div class="a--b-bbaa-bacdd-1j3xh4d e1mv6sxx0"><svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" width="18" height="18" role="img" aria-hidden="true" focusable="false"><path d="M17.5 11.6L12 16l-5.5-4.4.9-1.2L12 14l4.5-3.6 1 1.2z"></path></svg></div></span><div aria-hidden="true" class="components-input-control__backdrop a--b-bbaa-bacdd-29yhbg em5sgkm2"></div></div></div>
```

**Comment by bobbingwide** [comment]
That's better.
![image](https://user-images.githubusercontent.com/2474435/150116680-f5ed2598-afd6-4fb3-a88b-0b180a8c45f8.png)

**Comment by bobbingwide** [comment]
Pragmatic solution  - remove the select list from the block. Just use the Settings.

Developed in oik v4.6.0. Released in oik v4.7.1
